apps/src/templates/progress/progressStyles.js

```diff
@@ -64,6 +77,216 @@ export const hoverStyle = {
     'background-color .2s ease-out, border-color .2s ease-out, color .2s ease-out'
 };
 
+/**
+ * ======================================
+ * Bubble styles
+ * ======================================
+ */
+
+export const BubbleSize = makeEnum('dot', 'letter', 'full');
```

Comment:
For my education, where do we use 'letter' vs. 'full' size?

Reply:
'letter' was introduced in the table refactor work. we didn't used to show the letters in the dots for sublevels, but i proposed adding that since i find it super helpful for bubble choice with lots of sublevels. the proposal was approved but required making the dots slightly bigger. so 'dot' == "dot in header (empty)", 'leter' == "dot in table with letter", and 'full' == "full-sized bubble". i will document this here in subsequent PR.

Reply:
Ah I see, so the 'letter' size only applies to sublevels?